src/analytics.js

```diff
@@ -56,3 +57,32 @@ export function triggerAnalyticsEvent(nodeOrDoc, eventType, opt_vars) {
     analytics.triggerEvent(eventType, opt_vars);
   });
 }
+
+/**
+ * Method to create scoped analytics element for any element.
+ * @param {!Element} parentElement
+ * @param {!JSONType} config
+ * @param {boolean=} loadAnalytics
+ */
+export function insertAnalyticsElement(
+    parentElement, config, loadAnalytics = false) {
+  const analyticsElem =
+      parentElement.ownerDocument.createElement('amp-analytics');
+  analyticsElem.CONFIG = config;
```

Comment:
I'd still like to insert `<script type="application/json">{json}</script>` into DOM. It's not used downstream, but still good to show in case anyone wants to see what actually gets reported. In other words, `analuyticsElement.CONFIG` is simply an optimization.